lib/Saml2/Utils.php

```diff
@@ -971,7 +971,7 @@ public static function deleteLocalSession()
             session_destroy();
         }
 
-        unset($_SESSION);
+        $_SESSION = [];
```

Comment:
I think PHP has a method for this: `session_unset`, see: https://www.php.net/manual/en/function.session-unset.php

They even left a note about how it's really wrong to unset the `$_SESSION` variable entirely.

Reply:
Good one, I checked this function out, yet it didn't work for me.
However I called the function after the `session_destroy` function was called.
It seems that doesn't work.

I'll adjust the pully